liborbum/src/Controller/Ee/Vpu/Vu/Interpreter/CVuInterpreter_FLOAT.cpp

```diff
@@ -779,50 +1517,106 @@ void CVuInterpreter::MINIbc_3(VuUnit_Base* unit, const VuInstruction inst)
 
 void CVuInterpreter::OPMULA(VuUnit_Base* unit, const VuInstruction inst)
 {
-    // TODO: Implement.
-#if defined(BUILD_DEBUG)
-    BOOST_LOG(Core::get_logger()) << boost::format("(%s, %d) OPMULA: Not implemented.") % __FILENAME__ % __LINE__;
-#else
-    throw std::runtime_error("OPMULA: Not implemented.");
-#endif
+    SizedQwordRegister& fs = unit->vf[inst.fs()];
```

Comment:
ok

Reply:
hmm... I took a look at both PCSX2 and DobieStation, and apparently the MAC.w flag is cleared in Dobie, but left as-is in PCSX2.

Reply:
really sorry, ran out of time on this - i will check this weekend coming.

interesting that they are different...